testnets/_IBC/osmosistestnet4-prysmdevnet.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "../../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "osmosistestnet4",
```

Comment:
you should know that osmosistestnet4 isn't really used anymore, and people are generally using osmosistestnet (osmo-test-5) for testnet transactions. I'll double check to see if osmosistestnet4 has actually been halted, but I suspect this was the wrong one.